### DISCUSSION TITLE
Dithering

### DISCUSSION THREAD

**Discussion by thatcher-gaming** [dlemstra/magick-wasm#139] [Q&A] [answered]
Is it possible to dither an image with this library? I'm looking to do something like this:

```
magick my-image.png -dither FloydSteinberg dithered.gif
```

**Comment by dlemstra** [comment, marked as answer]
There was no support for that but in the next release you will be able to do this with the `quantize` method of `MagickImage`.

**Reply by thatcher-gaming** [reply]
thanks!

**Reply by dlemstra** [reply]
That release has already been published.
